lib/collective-sections.js

```diff
@@ -217,6 +217,9 @@ const getSectionsToRemoveForUser = (collective, isAdmin) => {
   if (isEmptyCollectiveLocation(collective)) {
     toRemove.add(Sections.LOCATION);
   }
+  if (collective.coreContributors?.length === 0) {
```

Comment:
That's not doing what we want, It should be visible if there are financial contributors for example.

Reply:
@znarf : Ops, I didn't think about the financial contributors. I've modified the check. Thanks for pointing it out. 😄